docker-compose.yml

```diff
@@ -400,6 +400,13 @@ services:
         image: "${DOCKER_USERNAME}/clang9-x86:${DOCKER_BUILD_TAG}"
         container_name: clang9-x86
         tty: true
+    clang10:
+        build:
+            context: clang_10
+            dockerfile: Dockerfile
+        image: "${DOCKER_USERNAME}/clang10:${DOCKER_BUILD_TAG}"
+        container_name: clang10
+        tty: true
```

Comment:
```suggestion
        tty: true
    clang10-x86:
        build:
            context: clang_10-x86
            dockerfile: Dockerfile
        image: "${DOCKER_USERNAME}/clang10-x86:${DOCKER_BUILD_TAG}"
        container_name: clang10-x86
        tty: true
    clang10-jnlp-slave:
        build:
            context: jenkins-jnlp-slave
            dockerfile: Dockerfile
            args:
                SOURCE_CONANIO_IMAGE: conanio/clang10
        image: "${DOCKER_USERNAME}/clang10-jnlp-slave:${DOCKER_BUILD_TAG}"
        container_name: clang10-jnlp-slave
        tty: true
    clang10-jnlp-slave-x86:
        build:
            context: jenkins-jnlp-slave
            dockerfile: Dockerfile
            args:
                SOURCE_CONANIO_IMAGE: conanio/clang10-x86
        image: "${DOCKER_USERNAME}/clang10-jnlp-slave-x86:${DOCKER_BUILD_TAG}"
        container_name: clang10-jnlp-slave-x86
        tty: true
```

Reply:
Does this Jenkins thing really require a 32 bit container? I was hoping we do not need this anymore for new compiler versions

Reply:
Yes, it's internal requirement for JFrog I think. But yeah, x86 it's a pain.

Reply:
Who can clarify the requirements for this? @lasote maybe?

Reply:
Maybe @danimtb or @jgsogo have that information.

Reply:
Having a look at the first PR for these images (https://github.com/conan-io/conan-docker-tools/pull/120), probably @maheshjfrog can say something about it.

Reply:
I am not familiar with requirements for individual conanio/* docker images (like the choice of 32 bit containers).
My support was to add jnlp slave jar to all existing conanio docker images so our jenkins pipeline can use them as nodes.

Reply:
I see, then we need all these `jnlp` _duplicated_ nodes.

Reply:
Thanks for the information

Based on everything, I'm assuming that we don't need 32-bit containers for new compiler versions anymore. And if so, we could still add them later on